README.md

```diff
@@ -197,19 +197,19 @@ Given a datetime or date, return a human-friendly representation
 of the amount of time difference, relative to the current time.
 
 ```clojure
-user> (require '[clj-time.core :as t])
-nil
+user> (import '(java.time LocalDateTime) '(java.time.temporal ChronoUnit))
+java.time.temporal.ChronoUnit
```

Comment:
Probably redundant?

Reply:
Both of these imports are used so I'm not sure why you think this is redundant?